### PR TITLE
Daniel/multipleexamples

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: event-schemas/examples/v1/m.room.member#third_party_invite [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "signatures": {
+          "magic.forest": {
+            "ed25519:3": "fQpGIW1Snz+pwLZu6sTy2aHy/DYWWTspTJRPyNp0PKkymfIsNffysMl6ObMMFdIJhk6g6pwlIqZ54rxo8SLmAg"
+          }
+        },
+        "token": "abc123"
+      }
+    }
+  },
+  "invite_room_state": [
```
Comment (Kegsay) [inline event-schemas/examples/v1/m.room.member#third_party_invite]: You're muxxing `invite_room_state` with `third_party_invite`. Can we have an example dedicated to `invite_room_state` please?

**Comment by Kegsay** [comment]
It looks like you branched this from #136, so that needs to land first really.

File: event-schemas/examples/v1/m.room.member#third_party_invite (hunk 1 above)
Comment (illicitonion) [inline event-schemas/examples/v1/m.room.member#third_party_invite]: Done

**Comment by illicitonion** [comment]
PTAL

**Comment by Kegsay** [comment]
http://matrix.org/speculator/diff/html/137#m-room-message is dumping all the `msgtype` values too :(

http://matrix.org/speculator/diff/html/137#m-text is doing something weird...

**Comment by illicitonion** [comment]
Left all of the msgtype examples.

Fixed the m-text issue.

**Comment by Kegsay** [comment]
LGTM

**Pull request merged**